From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Mangeonjean?= <loic@coderpad.io>
Date: Thu, 25 Apr 2024 19:14:25 +0200
Subject: [PATCH] fix: change syntax that language service refactor doesn't
 support well

the re-exports source are removed during "Move to new file" refactor
---
 src/vs/workbench/contrib/tasks/common/taskService.ts | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/src/vs/workbench/contrib/tasks/common/taskService.ts b/src/vs/workbench/contrib/tasks/common/taskService.ts
index 10e3cc0907e..ee76bd74989 100644
--- a/src/vs/workbench/contrib/tasks/common/taskService.ts
+++ b/src/vs/workbench/contrib/tasks/common/taskService.ts
@@ -15,7 +15,10 @@ import { IStringDictionary } from '../../../../base/common/collections.js';
 import { RawContextKey, ContextKeyExpr } from '../../../../platform/contextkey/common/contextkey.js';
 import { URI } from '../../../../base/common/uri.js';
 import { IMarkerData } from '../../../../platform/markers/common/markers.js';
-export type { ITaskSummary, Task, ITaskTerminateResponse as TaskTerminateResponse };
+
+export type { Task } from './tasks.js';
+export type { ITaskSummary, ITaskTerminateResponse as TaskTerminateResponse } from './taskSystem.js';
+
 export const CustomExecutionSupportedContext = new RawContextKey<boolean>('customExecutionSupported', false, nls.localize('tasks.customExecutionSupported', "Whether CustomExecution tasks are supported. Consider using in the when clause of a \'taskDefinition\' contribution."));
 export const ShellExecutionSupportedContext = new RawContextKey<boolean>('shellExecutionSupported', false, nls.localize('tasks.shellExecutionSupported', "Whether ShellExecution tasks are supported. Consider using in the when clause of a \'taskDefinition\' contribution."));
 export const TaskCommandsRegistered = new RawContextKey<boolean>('taskCommandsRegistered', false, nls.localize('tasks.taskCommandsRegistered', "Whether the task commands have been registered yet"));
